specification/cdn/resource-manager/Microsoft.Cdn/stable/2021-06-01/cdn.json

```diff
@@ -6428,6 +6476,10 @@
           "items": {
             "$ref": "#/definitions/ErrorDetail"
           },
+          "x-ms-identifiers": [
+            "message",
```

Comment:
Does "target" need "message" to form a composite identifier? Would "target" by itself not be enough in this case?